roles/common/tasks/ssh.yml

```diff
@@ -9,12 +9,15 @@
     state: directory
 
 - name: Set authorized keys
-  template:
-    src: templates/ssh/{{ inventory_hostname }}/authorized_keys.j2
-    dest: /root/.ssh/authorized_keys
-    owner: root
-    mode: 0600
+  authorized_key:
```

Comment:
I kind of like this way, since it will sync up with keys as they rotate. But if you dislike it @ovv I can revert to the manual way